presto-docs/src/main/sphinx/release/release-0.114.rst

```diff
@@ -7,3 +7,8 @@ General Changes
 
 * Fix ``%k`` specifier for :func:`date_format` and :func:`date_parse`.
   It previously used ``24`` rather than ``0`` for the midnight hour.
+
+Hive Changes
+------------
+
+* Fix ORC reader for Hive connector.
```

Comment:
The "Hive connector" part is redundant since it is in the "Hive changes" section.